admin/github.go

```diff
@@ -56,6 +59,13 @@ type githubClient struct {
 	installationCache *simplelru.LRU
 }
 
+func installationCacheKey(installationID int64, repoID *int64) string {
+	if repoID != nil {
+		return fmt.Sprintf("%d-%d", installationID, *repoID)
+	}
+	return fmt.Sprintf("%d", installationID)
+}
```

Comment:
nit: put util functions at the end of the file: https://github.com/uber-go/guide/blob/master/style.md#function-grouping-and-ordering